devnet-sdk/system/periphery/go-ethereum/fees.go

```diff
@@ -27,18 +27,29 @@ type EIP1559FeeEstimator struct {
 	// Access to the Ethereum client is needed to get the fee information from the chain
 	client EIP1159FeeEthClient
 
+	// The tip multiplier is used to increase the maxFeePerGas (GasFeeCap) by a factor
```

Comment:
is this comment correct? it's referring to the tip when its on the base line.

Reply:
Done fixed